### DISCUSSION TITLE
Switch to HTTPS

### DISCUSSION THREAD

**Discussion by alessandromas** [AbsaOSS/spline#977] [Q&A]
I'm actually running the 0.5.6 version of Spline installed via WAR deployment on Tomcat ( CentOS )

For security reason i need to know if is it possibile to use the HTTPS protocol instead of HTTP mainly for rest gateway. A nice have would be to have the same switch for GUI and Agent

Is there a way? If so, how can i do that?

**Comment by wajda** [comment]
Yes, of course it's possible and is very easy to do. Being a standard Java web application Spline has nothing to do with handling transport protocols - it's managed by the web server. If you are deploying WARs to a Tomcat all you need to do is to generate a certificate (unless you already have it) and uncomment a few lines in the `server.xml` configuration file. Refer the [Tomcat documentation](https://tomcat.apache.org/tomcat-9.0-doc/ssl-howto.html)
